Administrator/src/ssw_pers_admin_common.c

```diff
@@ -595,7 +595,7 @@ sint_t persadmin_compress(pstr_t compressTo, pstr_t compressFrom)
     }
 
     memset(pchParentPath, 0, sizeof(pchParentPath));
-    snprintf(pchParentPath, sizeof(pchParentPath), compressFrom);
+    snprintf(pchParentPath, "%s", sizeof(pchParentPath), compressFrom);
```

Comment:
Good and important fix to make.  But when I looked at the details I got confused.

If snprintf has the following prototype:
```
int snprintf(char *str, size_t size, const char *format, ...);
```
then aren't these arguments in the wrong order?

Does it compile?